DeezyMatch/tests/test_utils.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+import pytest
+
+from DeezyMatch import utils
+
+def test_string_split():
+
+    # -------------------
+    kwds = {"x": "py 001 $  ", 
+            "tokenize": ["char"], 
+            "prefix_suffix": ["|", "|"]}
+    assert utils.string_split(**kwds) == ['|', 'p', 'y', ' ', '0', '0', '1', ' ', '$', ' ', ' ', '|']
+    
+    # -------------------
+    kwds = {"x": "py 001 $  ", 
+            "tokenize": ["char"],  
+            "prefix_suffix": ["|", None]}
+    assert utils.string_split(**kwds) == ['p', 'y', ' ', '0', '0', '1', ' ', '$', ' ', ' ']
```

Comment:
Hi @kasra-hosseini, I would expect in this case the following output (i.e. a prefixed "|", no suffix):
```
['|', 'p', 'y', ' ', '0', '0', '1', ' ', '$', ' ', ' ']
```
Maybe this could be easily solved by translating None to empty string?

Reply:
Good point. I will change it in a sec.

Reply:
How does it look? https://github.com/Living-with-machines/DeezyMatch/pull/111/commits/63889476b0c0239c22a7c88dfbe3e75519c81988